### PR TITLE
bug 1145793 - Remove references to old redesign class

### PR DESCRIPTION
**Pull request by darkwing**
We don't need this class anmore \o/

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Lines 1609-1629 of the customcss.style file need to be updated as well.

**Comment by stephaniehobson** [comment]
Glad to see this going! We need to update the sphinx templates to remove the class as well but we can create a separate bug for that, it's not merge blocking.

**Comment by darkwing** [comment]
Good call Stephanie!  Updated!

File: media/stylus/components/wiki/customcss.styl [hunk 1]
```diff
@@ -909,23 +909,12 @@ dl {
 .indicatorInHeadline,
 .blockIndicator,
 .overheadIndicator {
-/*box-shadow: 1px 1px 1px #CCC*/
     margin: 0px 0px 1.5em;
     border: 5px solid;
-    padding: 10px 10px 0px 10px;
+    padding: 10px;
```
Comment (stephaniehobson) [inline media/stylus/components/wiki/customcss.styl]: These styles are being over-ridden by a declaration on line 948 and the inlineIndicators don't look right anymore. Before the added specificity of the redesign class was preventing this conflict.

I suspect but am not certain that we can just remove inlineIndicator from the other declaration.

Good testing page for this: https://developer.mozilla.org/en-US/docs/User:teoli/In_content

Comment (stephaniehobson) [inline media/stylus/components/wiki/customcss.styl]: Looks like it's also a problem with `.overheadIndicator` when used in combination with `.standardNoteBlock` from line 979.

**Pull request closed without merging**